docs/_toc.yml

```
@@ -21,6 +21,8 @@ parts:
       title: MIDC
     - file: station_network_solrad
       title: SOLRAD
+    - file: station_network_surfrad
+      title: SURFRAD
     - file: station_network_srml
       title: SRML
     - file: station_network_esmap
```

```suggestion
    - file: station_network_srml
      title: SRML
    - file: station_network_surfrad
      title: SURFRAD
    - file: station_network_esmap
```
List networks in alphabetical order.